packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageMarketCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { useDynamicLeverage } from '@deriv/api';
+import { WalletText } from '../../../../components/Base';
+import { DynamicLeverageTableColumnHeader } from './DynamicLeverageTableColumnHeader';
+import './DynamicLeverageMarketCard.scss';
+
+type TDynamicLeverageMarketCardProps = NonNullable<
+    Exclude<ReturnType<typeof useDynamicLeverage>['data'][0], 'undefined'>
```

Comment:
If possible we can add this type into the `types.ts` file under the `THooks` namespace

Reply:
Yeaps, please add it there

Reply:
Updated.